src/cmd/tools/m3db_tool/main/database/cmd.go

```diff
@@ -0,0 +1,65 @@
+package database
+
+import (
+	"flag"
+	"fmt"
+	"go.uber.org/zap"
+	"os"
+
+	"github.com/m3db/m3/src/cmd/tools/m3db_tool/main/common"
+	"github.com/m3db/m3/src/cmd/tools/m3db_tool/main/flagvar"
+	"github.com/m3db/m3/src/query/generated/proto/admin"
+)
+
+var (
+	CmdFlags   *flag.FlagSet
+	createYaml = flagvar.File{}
+
+	defaultPath = "/api/v1/database"
+	debugQS     = "debug=true"
+)
+
+func init() {
+	CmdFlags = flag.NewFlagSet("db", flag.ExitOnError)
+	CmdFlags.Var(&createYaml, "create", "Path to yaml for simplified db creation with sane defaults.")
+	CmdFlags.Usage = func() {
+		fmt.Fprintf(CmdFlags.Output(), `
+This is the "%s" subcommand for database scoped operations.
+
+Description:
+
+This subcommand allows the creation of a new database from a yaml specification.
+
+Usage of %s:
+
+`, CmdFlags.Name(), CmdFlags.Name())
+		CmdFlags.PrintDefaults()
+	}
+}
+
+func Cmd(log *zap.SugaredLogger) {
+
+	if err := CmdFlags.Parse(flag.Args()[1:]); err != nil {
+		CmdFlags.Usage()
+		os.Exit(1)
+	}
+
```

Comment:
(nitpick) all these empty lines are not required in Go unless you are staring something new. Check code of stdlib for reference.